app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionItemHolder.kt

```diff
@@ -64,25 +61,9 @@ class ConnectionItemHolder(parent: ViewGroup, private val listener: ListItemClic
         else listItemView.setBackgroundColor(bgColor)
 
         titleView.text = item.name
-
-        val description = if (item.consentsCount.isEmpty()) {
-            SpannableStringBuilder(item.statusDescription)
-        } else {
-            SpannableStringBuilder("${item.consentsCount} \u00B7 ${item.statusDescription}")
-        }
-
-        description.setSpan(
-            robotoMedium?.style?.let { StyleSpan(it) } ?: return,
-            0,
-            item.consentsCount.length,
-            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
-        )
-        description.setSpan(
-            robotoRegular?.style?.let { StyleSpan(it) } ?: return,
-            description.indexOf(item.statusDescription, 0),
-            description.length,
-            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
-        )
-        subTitleView.setText(description, TextView.BufferType.SPANNABLE)
+        subTitleView.text = SpannableStringBuilder()
+            .appendFacedText(item.consentsDescription, R.font.roboto_medium, subTitleView.context)
```

Comment:
where u set `roboto_regular` font?

Reply:
if we not apply font face in Spannable then fontface from textview style will be applied.